apis/duck/v1/status_types.go

```diff
@@ -99,17 +99,32 @@ func (s *Status) GetCondition(t apis.ConditionType) *apis.Condition {
 }
 
 // ConvertTo helps implement apis.Convertible for types embedding this Status.
-func (source *Status) ConvertTo(ctx context.Context, sink *Status) {
+//
+// By default apis.ConditionReady and apis.ConditionSucceeded will be copied over to the
+// sink. Other conditions types are tested against a list of predicates. If any of the predicates
+// return true the condition type will be copied to the sink
+func (source *Status) ConvertTo(ctx context.Context, sink *Status, predicates ...func(apis.ConditionType) bool) {
```

Comment:
I'd rather take a single predicate and leave conjunction/disjunction semantics up to the caller?

Reply:
the `...` is there to ensure this isn't a breaking change

Reply:
but it has that side effect of being a disjunction

Reply:
🤔 I like that even better, thanks.